grails-app/services/org/pih/warehouse/shipping/ShipmentService.groovy

```diff
@@ -2155,4 +2156,28 @@ class ShipmentService {
             throw new ValidationException("Invalid shipment", shipment.errors)
         }
     }
+
+    Container initializeContainerFromShipment(Shipment shipment){
```

Comment:
Let's move this into Shipment and call the method. There's actually already one there that takes a ContainerType argument.
```
Container addNewContainer() { }
```